File: prisioner/java/src/main/java/com/everis/competition/jail/PrisonerAlwaysNegate.java
package com.everis.competition.jail;

import java.util.ArrayList;

import com.everis.competition.prisoner.Decision;
import com.everis.competition.prisoner.Prisoner;

    public class PrisonerAlwaysNegate extends Prisoner {

        public PrisonerAlwaysNegate(String name) {
            super(name);
        }

        @Override
        public int decide(ArrayList<Decision> decisions) {
            return Decision.NEGATION;
        }
    }

File: prisioner/js/jail/PrisonerAlwaysNegate.js
var Prisoner = require('../prisoner/Prisoner').Prisoner;
var DECISION_OPTIONS = require('../prisoner/Decision').DECISION_OPTIONS;
var Decision = require('../prisoner/Decision').Decision;

var PrisonerAlwaysNegate = function(name) {
    Prisoner.call(this, name)

}
PrisonerAlwaysNegate.prototype = Object.create(Prisoner.prototype);
Object.defineProperty(PrisonerAlwaysNegate.prototype, 'constructor', {
    value: PrisonerAlwaysNegate,
    enumerable: false,
    writable: true
});
PrisonerAlwaysNegate.prototype.decide = function(decisions){
    return DECISION_OPTIONS.NEGATION;
}

module.exports.PrisonerAlwaysNegate = PrisonerAlwaysNegate;